src/site/layouts/health_care_region_patient_family_services_page.drupal.liquid

```diff
@@ -0,0 +1,50 @@
+{% include "src/site/includes/header.html" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+<div id="content" class="interior">
+    <main class="va-l-detail-page">
+        <div class="usa-grid usa-grid-full">
+
+            {% include 'src/site/navigation/facility_sidebar_nav.drupal.liquid' with facilitySidebar %}
+
+            <div class="usa-width-three-fourths">
+                <h1>Patient & Family Services</h1>
+                <div class="va-introtext">
+                    {{fieldPatientFamilyServicesIn.processed }}
+                </div>
+
+                {% capture difference %} {{ careCoordinatorPatientFamilyServices.entities | size | minus:1 }} {% endcapture %}
+                {% unless difference contains '-' %}
+                    <section class="care-patient-family-services">
+                        <h2>Care Coordinator</h2>
+                        {% for careService in careCoordinatorPatientFamilyServices.entities %}
+                            {% include "src/site/facilities/health_service.drupal.liquid" with healthService = careService %}
+                        {% endfor %}
+                    </section>
+                {% endunless %}
+
+                {% capture difference %} {{ socialProgramsPatientFamilyServices.entities | size | minus:1 }} {% endcapture %}
+                {% unless difference contains '-' %}
+                    <section class="social-patient-family-service">
+                        <h2>Social Programs and Services</h2>
+                        {% for socialService in socialProgramsPatientFamilyServices.entities %}
+                            {% include "src/site/facilities/health_service.drupal.liquid" with healthService = socialService %}
+                        {% endfor %}
+                    </section>
+                {% endunless %}
+
+                {% capture difference %} {{ healthWellnessPatientFamilyServices.entities | size | minus:1 }} {% endcapture %}
+                {% unless difference contains '-' %}
```

Comment:
Could this be shortened to `{% healthWellnessPatientFamilyServices.entities != empty %}`, or is it doing something different with the `capture`?

Reply:
Testing for empty doesn't work on arrays. You have to check the size.